packages/syft/src/syft/core/tensor/autodp/gamma_tensor.py

```diff
@@ -2381,6 +2382,188 @@ def _squeeze(state: dict) -> jax.numpy.DeviceArray:
             sources=output_state,
         )
 
+    def any(
+        self,
+        axis: Optional[Union[int, Tuple[int, ...]]] = None,
+        keepdims: Optional[bool] = False,
+        where: Optional[ArrayLike] = None,
+    ) -> GammaTensor:
+        func = "any"
+
+        def _any(state: dict) -> jax.numpy.DeviceArray:
+            return jnp.any(
+                *[
+                    i.reconstruct() if isinstance(i, GammaTensor) else i
+                    for i in state.values()
+                ]
+            )
+
+        output_state = dict()
+        output_state[self.id] = self
+
+        mapper[func] = _any
```

Comment:
If we move _any() out as well, then we don't have to set and re-set this lookup table. We could stead just create this when syft is imported.

Reply:
(ditto for other methods)

Reply:
yep sounds good

Reply:
fix